server/index.js

```diff
@@ -1,6 +1,6 @@
-// Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
-// In particular, applicationinsights automatically collects bunyan logs
-const appInsightsClient = require('./utils/azure-appinsights')()
+import buildAppInsightsClient from './utils/azure-appinsights'
+import createOffenderService from './services/offenderService'
+import createReportingService from './services/reportingService'
```

Comment:
👍 sweet, I thought you could mix the 2 import styles but got into a bit of a pickle with linting rules